ts/navigation/routes.ts

```diff
@@ -46,12 +46,15 @@ const ROUTES = {
 
   // Main
   MAIN: "MAIN",
-  MAIN_PROFILE: "MAIN_PROFILE",
 
   // Messages
   MESSAGES_HOME: "MESSAGES_HOME",
   MESSAGES_LIST: "MESSAGES_LIST",
-  MESSAGE_DETAILS: "MESSAGE_DETAILS"
+  MESSAGE_DETAILS: "MESSAGE_DETAILS",
+
+  // Profile
+  PROFILE_HOME: "PROFILE_HOME",
```

Comment:
I can't tell the difference between `HOME` and `MAIN`, they both kind of refer to the "root" - perhaps better naming could help?

Reply:
Going to rename with PROFILE_NAVIGATOR and PROFILE_MAIN.